libpod/container_internal.go

```diff
@@ -683,6 +684,21 @@ func (c *Container) makeBindMounts() error {
 		c.state.BindMounts["/run/.containerenv"] = containerenvPath
 	}
 
+	// Add secrets mounts
+	mountsFiles := []string{secrets.OverrideMountsFile, secrets.DefaultMountsFile}
+	for _, file := range mountsFiles {
+		secretMounts, err := secrets.SecretMounts(file, c.config.MountLabel, c.state.RunDir)
+		if err != nil {
+			logrus.Warn("error mounting secrets, skipping...: %s", err)
+			continue
+		}
+		for _, mount := range secretMounts {
```

Comment:
This will out our secrets RW instead of RO like @umohnani8 had in her patch. If that's a problem, we can extend BindMounts in the DB to include an RO/RW bool.